frontend/javascripts/test/sagas/save_saga.spec.js

```diff
@@ -103,7 +103,7 @@ test("SaveSaga should send update actions", t => {
   saga.next(); // select state
   expectValueDeepEqual(t, saga.next([]), take("PUSH_SAVE_QUEUE_TRANSACTION"));
   saga.next(); // race
-  saga.next(SaveActions.pushSaveQueueTransaction(updateActions));
```

Comment:
The previous argument here was wrong as this `saga.next` corresponds to the `race` of the `save_saga.js` in lines 667ff. This race only takes a timeout or a `saveNowAction`. In order for the test to work with the changes of this pr, I had to correct this.